src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -812,13 +812,17 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
             $dumper->setProxyDumper(new ProxyDumper(substr(hash('sha256', $cache->getPath()), 0, 7)));
         }
 
+        if ($this->loadClassCache || class_exists(ClassCollectionLoader::class, false)) {
+            // if legacy inlining has been used, disable container inlining
+            $container->getParameter('container.dumper.inline_class_loader', false);
```

Comment:
```diff
- getParameter
+ setParameter
```

? But that's a `FrozenParameterBag` here

Reply:
good catch, should be better now